packages/channels/src/vonage/config.ts

```diff
@@ -0,0 +1,16 @@
+import Joi from 'joi'
+import { ChannelConfig } from '../base/config'
+
+export interface VonageConfig extends ChannelConfig {
+  apiKey: string
+  apiSecret: string
+  signatureSecret: string
+  useTestingApi?: boolean
```

Comment:
I didn't find a reason yet to require the applicationId and the privateKey here so I removed them